src/commands/deployWorkspaceProject/deploymentConfiguration/workspace/DeploymentConfigurationListStep.ts

```diff
@@ -38,7 +39,7 @@ export class DeploymentConfigurationListStep extends AzureWizardPromptStep<Works
 
         return {
             executeSteps: [
-                // Todo: Shallow (local fs) validation step(s)
+                new FilePathsVerifyStep(),
```

Comment:
It may be a bit redundant, but should this be broken into three separate steps?

The main reason being it would easier to exclude looking for a dockerfile path in the future. It could also be easier/more streamlined to tell the user _which_ one failed, etc. 

It'd be easy enough to create a base step and then just extend that since all of the logic should nearly be identical.

Reply:
Yeah that works too, can we merge this step for now so I can keep testing the logic in later PRs?  Megan can just remove this step in her PR(s) later.

Reply:
I might also re-explore exploding out the container app verifications then to see if we can make it a flat list.  I'll probably need to make two steps instead of one.